app/models/viewer.rb

```diff
@@ -0,0 +1,10 @@
+##
+# Model for viewer
+class Viewer < ActiveRecord::Base
+  belongs_to :exhibit, class_name: Spotlight::Exhibit
+
+  def to_partial_path
+    return 'oembed_default' if viewer_type == 'sul-embed'
+    "../viewers/#{viewer_type}"
```

Comment:
I'm surprised that we need the `../` here.  Did `"viewers/#{viewer_type}" or `"viewers/_#{viewer_type}"` not work?